src/components/heading/Header.js

```diff
@@ -0,0 +1,86 @@
+import "./style.css";
+import React, {useContext,useState} from 'react';
+import { AppBar, IconButton, Toolbar, Typography, FormControl, InputLabel, Button } from "@mui/material"
+import Search from '../../Search';
+import { Roofing } from "@mui/icons-material"
+import Navbar from "../../components/navbar/Navbar";
+import ApartsFilter from "./ApartsFilter";
+import RoomsFilter from "./RoomsFilter";
+import PriceFilter from "./PriceFilter";
+import Home from "../../pages/Home";
+import { red } from "@mui/material/colors";
+
+
+
+function Header() {
+  return (
+    <>
+
+      <AppBar className="appbar" position="static" sx={{backgroundColor: '#b87f59'}} >
```

Comment:
Please read about `mui` pallete and use colors from it(you can add your custom colors to the palette) 

https://mui.com/material-ui/customization/palette/